app/views/home/_navbar.html.haml

```diff
@@ -13,7 +13,7 @@
     
     .flex.items-center.gap-2
       - if user_signed_in?
-        .btn-primary= link_to "#{current_user.email}", profile_path(current_user)
+        .btn-primary= link_to "Hi, #{current_user.username}", profile_path(current_user)
```

Comment:
The essence of this change is to make the navbar look better some. It doesn't seem okay for it to just display the user's email address